Open_Science_New

```diff
@@ -384,18 +369,18 @@
 	      <h2><a href="">Hackathons</a></h2>
 	      <tr>
 		<td width="75%" style="vertical-align: middle">
-		  <img src="images/C-BASE.png">
+		  <a href="http://www.digitalinberlin.de/c-base/"><img src="images/C-BASE.png"></a>
```

Comment:
let's remove this link.  That pic was just the uber-cool venue for one particular hackathon in 2014, no need to point people to it.

Reply:
done!